lib/tlog/tap.c

```diff
@@ -372,6 +428,17 @@ tlog_tap_teardown(struct tlog_errs **perrs,
             return grc;
         }
         tap->pid = 0;
+        if (tap->ut.ut_type == USER_PROCESS) {
```

Comment:
Do we need to make sure tap->ut is non-NULL here?

Reply:
No, because tap->ut was not a pointer, but I'm changing this.